### PR TITLE
Make digital traces match memmap dtypes in neo.

### PR DESCRIPTION
**Pull request by h-mayorquin**
@zm711

### CODE REVIEW

File: neo/rawio/intanrawio.py
Comment (h-mayorquin) [inline neo/rawio/intanrawio.py:490]: This is the only real change on this PR. All the other is a variable re-name to avoid further confusion.

**Comment by zm711** [comment]
This works for me once tests pass. You want it to be draft for more discussion on SI?

**Comment by h-mayorquin** [comment]
Na, man, your call as a neo mantainer.  I have a slight push to got for the performance improvement but I think you are right that this might better (and is easier). We should close the spikeinterface issue once we merge this.

As an aside, I never thought that it makes sense to run a test again when you mark something from draft to ready.

**Review by zm711 — APPROVED** [review]

**Pull request merged**
